nipype/utils/nipype_cmd.py

```diff
@@ -47,32 +47,31 @@ def add_options(parser=None, module=None, function=None):
 
 
 def run_instance(interface, options):
-    if interface:
-        print("setting function inputs")
-
-        for input_name, _ in list(interface.inputs.items()):
-            if getattr(options, input_name) != None:
-                value = getattr(options, input_name)
-                if not isinstance(value, bool):
-                    # traits cannot cast from string to float or int
-                    try:
-                        value = float(value)
-                    except:
-                        pass
-                    # try to cast string input to boolean
-                    try:
-                        value = str2bool(value)
-                    except:
-                        pass
+    print("setting function inputs")
```

Comment:
I think for the old nipype_cmd it makes sense. Probably it doesn't for nipype_display_crash and others.